### DISCUSSION TITLE
How to navigate `hover` info, and which keymaps should be configured?

### DISCUSSION THREAD

**Discussion by hopezh** [LazyVim/LazyVim#1899] [Q&A] [answered]
As shown below, when there is a lot of info shown in the pop-up window when using `K` to `hover` over the name of the module/class, which keymap should we use to navigate them up and down? and how to specify such keymap if there isn't? 

Thanks.

<img width="1235" alt="Screenshot 2023-10-29 at 18 36 23" src="https://github.com/LazyVim/LazyVim/assets/23435810/0aab9f89-4fa8-450c-8cb8-7809344337b1">

**Comment by hopezh** [comment, marked as answer]
Allow me to reply myself: pressing `K` **twice** to go into the window of hover info to scroll it. 

Source:
- https://www.reddit.com/r/neovim/comments/ksf9ve/comment/gik73uh/?utm_source=share&utm_medium=web2x&context=3

**Reply by folke** [reply]
You can also use `<c-f>` and `<c-b>` from the main window to scroll the hover

**Reply by elijahfhopp** [reply]
Noob question: What is the ideal/most correct/idiomatic way of exiting the hover window after entering with `K`x2? Switching windows back to the parent/containing window? Pressing `K` a third time seems like it's supposed to (giving an error about `man` because I am on Alpine Linux) jump to the docs.

**Reply by dpetka2001** [reply]
Just press `q`?

**Reply by elijahfhopp** [reply]
> Just press `q`?

*facepalm* that's it. Thanks, :heart:!
